homeassistant/components/knx/__init__.py

```diff
@@ -398,16 +406,16 @@ def register_callback(self) -> TelegramQueue.Callback:
         address_filters = list(
             map(AddressFilter, self.config[DOMAIN][CONF_KNX_EVENT_FILTER])
         )
-        return self.xknx.telegram_queue.register_telegram_received_cb(
+        return self.xknx.telegram_queue.register_telegram_received_cb(  # type: ignore[no-any-return]
```

Comment:
I have no idea why mypy handles this as `Any`. It returns a TelegramQueue.Callback in xknx code 
https://github.com/XKNX/xknx/blob/6a9be156132bac5233a0a5000e8ea953369d642d/xknx/core/telegram_queue.py#L80
I have tried changeing it to a forward reference from using only `-> Callback:` previously (xknx 0.17.2) but this didn't change anything.

Reply:
🤷